tests/unit_tests/test_lobbyconnection.py

```diff
@@ -255,3 +256,230 @@ def test_avatar_select_no_avatar(mock_query, fa_server_thread):
     with pytest.raises(KeyError):
         fa_server_thread.command_avatar({'action': 'select'})
 
+# handle action
+def test_handle_action_ping(fa_server_thread):
+    fa_server_thread.sendReply = mock.Mock()
+    fa_server_thread.handleAction('PING', mock.Mock())
+    fa_server_thread.sendReply.assert_called_once_with('PONG')
+
+def test_handle_action_pong(fa_server_thread):
+    assert fa_server_thread.ponged == False
+    fa_server_thread.handleAction('PONG', mock.Mock())
+    assert fa_server_thread.ponged == True
```

Comment:
I think this is a bad test.

`lobbyconnection.ponged` should for one, not exist (We should use the `SO_KEEPALIVE` option) and if it should exist it should be a timestamp.

We should either have a TODO here or redo the implementation now (I don't think it's high priority)